doc/user_guide/user_guide.md

```diff
@@ -0,0 +1,71 @@
+# User Guide
+
+This user guide helps you with getting started with the Files Virtual Schemas.
+
+### Installation
+ 
+Upload the latest available [release of this adapter](https://github.com/exasol/virtual-schema-common-document-files/releases) to BucketFS.
+See [Create a bucket in BucketFS](https://docs.exasol.com/administration/on-premise/bucketfs/create_new_bucket_in_bucketfs_service.htm) and [Upload the driver to BucketFS](https://docs.exasol.com/administration/on-premise/bucketfs/accessfiles.htm) for details.
+
+Then create a schema to hold the adapter script.
+
+```
+CREATE SCHEMA ADAPTER;
+```
+
+Next create the Adapter Script:
+ ```
+CREATE OR REPLACE JAVA ADAPTER SCRIPT ADAPTER.FILES_ADAPTER AS
+    %scriptclass com.exasol.adapter.RequestDispatcher;
+    %jar /buckets/bfsdefault/default/document-virtual-schema-dist-1.0.0-files-0.1.0.jar;
+/
+```
+
+In addition to the adapter script you need to create a UDF function that will handle the loading of the data:
+```
+CREATE OR REPLACE JAVA SET SCRIPT ADAPTER.IMPORT_FROM_DOCUMENT_FILES(
+  DOCUMENT_FETCHER VARCHAR(2000000),
+  REMOTE_TABLE_QUERY VARCHAR(2000000),
+  CONNECTION_NAME VARCHAR(500))
+  EMITS(...) AS
+    %scriptclass com.exasol.adapter.document.UdfRequestDispatcher;
+    %jar /buckets/bfsdefault/default/document-virtual-schema-dist-1.0.0-files-0.1.0.jar;
+/
+```
+
+## Creating a Virtual Schema
+ 
+Now you need to define a connection to place where the Files are stored:
+
+* BucketFS:
+     ```
+    CREATE CONNECTION BUCKETFS_CONNECTION
+        TO 'bucketfs:/bfsdefualt/default/'
+        USER ''
+        IDENTIFIED BY '';
+    ```
+  You can leave `USER` and `IDENTIFIED BY` empty.
+  
+  The path after `buketfs:` is the base path for the file names you define in the mapping definition.
+  The Adapter will concatenate both paths. 
```

Comment:
I did not understand what the `both` refers to? What is the second path?

Reply:
```suggestion
  The adapter will concatenate the base path from the connection and the path defined in the mapping definition. 
```

Reply:
This much better!